lib/inputs.rb

```diff
@@ -8,30 +8,23 @@ def self.yn(question)
       input = _input_evaluator.call
       input = input.chomp
     end until %w(y n).include?(input.chomp)
-
-    case input
-    when 'y'
-      true
-    when 'n'
-      false
-    end
+    input == 'y'
   end
 
-  def self.name(question)
+  def self.text(question)
     output question
-    name = _input_evaluator.call
-    name = name.chomp
-    output name.green
-    name
+    _input_evaluator.call.chomp
   end
+  alias_method :name, :text
 
-  def self.names(question)
+  def self.multiple_text(question)
```

Comment:
...but please don't change this in PR , feel free to keep it in your fork but wont be able to accept this  (due to backwards compatibility)